src/stan-users-guide/gaussian-processes.Rmd

```diff
@@ -1086,3 +1151,98 @@ generated quantities {
   Omega = L_Omega * L_Omega';
 }
 ```
+
+## Supported Covariance Functions
+
+Currently, Stan supports several build in covariance functions (kernels). These are the squared exponential, also known as the radial basis function or the gaussian kernel, a dot product or linear covariance function, the periodic covariance function, and three additional special cases of the Matern covariance function, that are the Matern 5/2 kernel, Matern 3/2 and the Matern 1/2 covariance function, or the exponential covariance function. We will review functional form, implementation and provide a visualization with simulated data below.
+
+### The Squared Exponential
+
+A common choice for a covariance function is the Squared Exponential covariance function. This is also known as the Radial Basis Function or the Gaussian kernel, due to the fact that its function resembles a gaussian. The functional form is as follows
```

Comment:
"Squared Exponential"
-> exponentiated quadratic
"the Radial Basis Function or the Gaussian kernel"
-> squared exponential, the radial basis function or the Gaussian kernel
"gaussian"
-> Gaussian
"is as follows"
-> is